crates/viewer/re_renderer/src/mesh.rs

```diff
@@ -200,6 +200,16 @@ pub struct GpuMesh {
     pub materials: SmallVec<[GpuMaterial; 1]>,
 }
 
+impl GpuMesh {
+    /// Returns the approximate byte size this `GpuMesh` contributes.
+    ///
+    /// The total buffer sizes are divided by how many strong references
+    /// they have.
```

Comment:
outdated, ended up not doing that because it's messy with the way we ref count these